frontend/src/modules/notification/models.ts

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from "react"
+import { AnyAction } from "redux"
+
+export const NOTIFICATION = 'NOTIFICATION'
+export const NOTIFICATION_REMOVE = `${NOTIFICATION}.REMOVE`
+
+export enum NotificationType {
+    SUCCESS = 'SUCCESS',
+    INFO = 'INFO',
+    WARNING = 'WARNING',
+    ERROR = 'ERROR',
+}
+
+export interface NotificationProps {
+    type: NotificationType
+    description: ReactNode
```

Comment:
Oops! You can't put ReactNodes in the redux store. I believe that you end up with an object containing a reference to the entire DOM and the redux serializer goes bananas trying to serialize the whole DOM. I think we will have to limit title and description to strings.